src/pymor/models/iosys.py

```diff
@@ -917,6 +919,153 @@ def hankel_norm(self, mu=None):
         """
         return self.hsv(mu=mu)[0]
 
+    def l2_norm(self, ast_pole_data=None, mu=None):
+        """Compute the L2-norm of the |LTIModel|.
+
+        Parameters
+        ----------
+        ast_pole_data
+            Can be:
+
+            - dictionary of parameters for :func:`~pymor.algorithms.eigs.eigs`,
+            - list of anti-stable eigenvalues (scalars),
+            - tuple `(lev, ew, rev)` where `ew` contains the anti-stable eigenvalues
+              and `lev` and `rev` are |VectorArrays| representing the eigenvectors.
+            - `None` if anti-stable eigenvalues should be computed via dense methods.
+        mu
+            |Parameter|.
+
+        Returns
+        -------
+        norm
+            L_2-norm.
+        """
+        if not isinstance(mu, Mu):
+            mu = self.parameters.parse(mu)
+        assert self.parameters.assert_compatible(mu)
+
+        A, B, C, D, E = (op.assemble(mu=mu) for op in [self.A, self.B, self.C, self.D, self.E])
+        options_lrcf = self.solver_options.get('lyap_lrcf') if self.solver_options else None
+
+        ast_spectrum = self.get_ast_spectrum(ast_pole_data, mu)
+
+        if len(ast_spectrum[0]) == 0:
+            return self.h2_norm()
+
+        K = bernoulli_stabilize(A, E, C.as_source_array(mu=mu), ast_spectrum, trans=False)
+        KC = LowRankOperator(K, np.eye(len(K)), C.as_source_array(mu=mu))
+
+        if not isinstance(D, ZeroOperator):
+            BmKD = B - LowRankOperator(K, np.eye(len(K)), D.as_source_array(mu=mu))
+        else:
+            BmKD = B
+
+        if self.dim_input <= self.dim_output:
+            cf = solve_lyap_lrcf(A - KC, E, BmKD.as_range_array(mu=mu),
+                                 trans=False, options=options_lrcf)
+            return np.sqrt(self.C.apply(cf, mu=mu).norm2().sum())
+        else:
+            of = solve_lyap_lrcf(A - KC, E, C.as_source_array(mu=mu),
+                                 trans=True, options=options_lrcf)
+            return np.sqrt(BmKD.apply_adjoint(of, mu=mu).norm2().sum())
+
+    @cached
+    def linf_norm(self, mu=None, return_fpeak=False, ab13dd_equilibrate=False):
+        """Compute the L_infinity-norm of the |LTIModel|.
```

Comment:
Maybe add an definition of the norm?

Reply:
And explain why this is the same as calling `hinf_norm`?

Reply:
Currently `LTIModel.hinf_norm` is based on slycot's [`ab13dd`](http://slicot.org/objects/software/shared/doc/AB13DD.html) which computes the L-infinity norm (there is no slycot function for H-infinity norm computation). For asymptotically stable systems H-inf and L-inf norms are identical, which is why we use it that way. It will probably make sense to put the current `hinf_norm` code into `linf_norm` method and have `hinf_norm` call  `linf_norm` rather than the other way around. Should we test for asymptotic stability in `hinf_norm` somehow such that it is more accurate?

Reply:
@pmli Do you have an opinion on this?

Reply:
I'm not sure, but since `hinf_norm` works only for dense systems, I guess it doesn't hurt too much to add a stability check using `eig`, and return `np.inf` if the system is not asymptotically stable.

Reply:
After doing some testing, it turned out that `eig` is very slow. E.g., for a 1000 x 1000 matrix `eig` takes roughly 60 seconds whereas in comparison `linf_norm` takes 0.0005 seconds. Since we have the note 'Assumes the system is asymptotically stable.' in the `hinf_norm` documentation, I would be for not adding the stability check for now.

Reply:
> It will probably make sense to put the current `hinf_norm` code into `linf_norm` method and have `hinf_norm` call `linf_norm` rather than the other way around. 

Agreed. Can you do that in this PR?

> Since we have the note 'Assumes the system is asymptotically stable.' in the hinf_norm documentation, I would be for not adding the stability check for now.

I'm fine with this. Maybe extend the documentation a little and say that under this assumption both norms are the same, so that we just defer to the other method?